MerlinAU.sh

```diff
@@ -1615,8 +1615,8 @@ _CreateEMailContent_()
         fwNewUpdateVersion="$(Get_Custom_Setting "FW_New_Update_Notification_Vers")"
    fi
 
-   # Remove "_rog" or "_tuf" suffix to avoid version comparison failures #
-   fwInstalledVersion="$(echo "$fwInstalledVersion" | sed 's/_\(rog\|tuf\)$//')"
+   # Remove "_rog" or "_tuf" or -gHASHVALUES suffix to avoid version comparison failures #
```

Comment:
Required for proper alpha comparisons to not fail. (Flashing alpha ontop of alpha)

Found here:

![image](https://github.com/user-attachments/assets/1c264b1f-d64e-4642-a8c2-5de268083537)